sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/src/Generated/ArchiveData.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.ResourceManager.ContainerRegistry.Models;
+using Azure.ResourceManager.Models;
+
+namespace Azure.ResourceManager.ContainerRegistry
+{
+    /// <summary>
+    /// A class representing the Archive data model.
+    /// An object that represents a archive for a container registry.
+    /// </summary>
+    public partial class ArchiveData : ResourceData
```

Comment:
The name is too generic. Consider appending RP prefix or renaming it.
- use `prepend-rp-prefix` to automatically append PR name. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L52-L57).
- use `rename-mapping` to rename a model. See example [here](https://github.com/Azure/azure-sdk-for-net/blob/176fd476588e8e61ebce747d66714cb39947cae5/sdk/azurestackhci/Azure.ResourceManager.Hci/src/autorest.md?plain=1#L58-L76)

Reply:
![image](https://github.com/user-attachments/assets/ae3fb886-21b3-460b-a584-58b5e0b88e5f)
 It seems like prepend-rp-prefix does not work for partial classes. I added in "AchiveData" according to the example and have received the error: "No Usage transform detected" Am I missing something?

Reply:
I also tried rename-mapping (ArchiveData: ContainerRegistryArchiveData" without success as well.

Reply:
I was able to mitigate this issue since I realized that "Data" is appended to the name, so I used prepend-rp-prefix with just "Archive"